guidelines/releases-guidelines/02-kyma-release-process.md

```diff
@@ -218,9 +213,12 @@ Ensure that the `prow/RELEASE_VERSION` file from the `test-infra` repository on
    2. Click the **generate new sheet** button.
    3. You will be asked for a GitHub personal access token. This token does not need any additional scopes!
 
-
 > **NOTE:** After the Kyma release is complete, proceed with [releasing Kyma CLI](/guidelines/releases-guidelines/03-kyma-cli-release-process.md).
 
 ## Post-release tasks
 
-Update `prow/RELEAS_VERSION` in the `master` branch of the `test-infra` repository with the name of the next minor release candidate and merge the pull request to `master`. For example, if the `RELEASE_VERSION` on the `master` branch is set to `1.4.2`, then change the version to `1.5.0-rc1`.
+1. Ask the Huskies team to upgrade the Kyma Helm chart on AppHub.
+
+        > **NOTE:** Because of a limitation on AppHub side, only a few people are allowed to create a PR, which currently includes the members of the Huskies team.
```

Comment:
```suggestion
        > **NOTE:** Because of a limitation on the AppHub side, only a few people are allowed to create such a PR, which currently includes the members of the Huskies team.
```